apps/website-25/src/components/lander/LandingPageBase.tsx

```diff
@@ -121,22 +121,33 @@ const LandingPageBase: React.FC<LandingPageBaseProps> = ({ hero, variant }) => {
             We started BlueDot to help others understand and work on challenges from emerging technologies.
           </p>
           <p className="mb-8 text-size-md">
-            Since 2021, we’ve designed our courses with some of the world’s leading experts and helped thousands of talented people build the skills to make a real difference.
+            Since 2021, we've designed our courses with some of the world's leading experts and helped thousands of talented people build the skills to make a real difference.
           </p>
           <CTALinkOrButton variant="secondary" url={ROUTES.about.url}>Read about us</CTALinkOrButton>
         </div>
-        <div className="flex justify-end -mt-16 pointer-events-none">
+        <div className="flex justify-end mt-0 md:-mt-16 pointer-events-none">
```

Comment:
tip: split out unrelated changes into different PRs - it reduces the chance of merge conflicts between people & means they can be merged independently (e.g. I'd be keen to merge this fix now, but need to wait until the other bits resolved)